.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "jsxSingleQuote": true,
-  "bracketSameLine": true
+  "bracketSameLine": true,
```

Comment:
I think setting this to false would result in slightly better visibility, let me know what everyone else thinks about this @Azzirr @bartosztrusinski

Reply:
Honestly I don't know how it works, but i found that against this (on true):
  return { name: "John", age: 30 };

we will have this (on false):
  return { 
    name: "John", 
    age: 30 
  };
  
  If yes - I agree with @nikkehtine, better visability is on false

Reply:
![image](https://github.com/bartosztrusinski/chit-chat/assets/33166095/417ffcbe-b213-4b69-9469-caf420ee3331)

this is the rule, closing > on separate line looks bad and bloats file but i can set it to false if everyone wants that

Reply:
okay, set this rule to `false`, merging